code-challenges.js

```diff
@@ -11,7 +11,18 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
+const checkTemp = (temp) => {
+  if(temp1 <= 212){
+    return `${temp1} is below boiling point`
+  }
+  else if(temp2 >= 212){
+    return `${temp2} is above boiling point`
+  }
+  else if(temp3 === 212){
+    return `${temp3} is at boiling point`
+  }
+}
+console.log(checkTemp(temp1))
```

Comment:
You're only checking a single variable into your console logs. It is SUPER important to be checking all of the given variables in your calls.